### PR TITLE
New structure and content for landing page

### PR DESCRIPTION
**Pull request by steren**
- Refactor SCSS by splitting each component into its scss file.
- Add more customer logos
- Add quotes
- tweak landing page styles: font size
- use Material Design button styles

Screenshot:
localhost-12345- 4

Follow up changes will:
- use variables instead of hard coded domain names
- add illustrations

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,194 +2,225 @@
...
+      <div class="row">
+        <div class="col-md-6">
+          <img src="{{site_root}}images/bazel-logo.svg"
+               title="Bazel"
+               alt="Bazel logo"
+               class="logo img-responsive center-block" />
+          <p class="tagline">{Fast, Correct} - Choose two</p>
+        </div>
+        <div class="col-md-6">
+          <h1>Build and test software of any size, quickly and reliably</h1>
```
Comment (davidzchen) [inline index.html]: The use of bold text for this title seems a bit out of place. How would it look if this used the same thinner font weight as the other headings on the website?

File: index.html [hunk 2]
```diff
@@ -2,194 +2,225 @@
...
-           build specification. On Linux, Bazel runs tools in a sandboxed
-           environment that contain only the minimum necessary files required.
-           Even tests are run in sandboxes, which ensures predictable,
-           environment-independent results.
-        </p>
+
+  <div class="landing-section summary">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-8 col-sm-12">
```
Comment (davidzchen) [inline index.html]: As @helenalt mentioned during our meeting Friday, this section seems a bit unbalanced. Can we try removing the `col-md-8`?

File: index.html (hunk 1 above)
Comment (steren) [inline index.html]: Done.

File: index.html (hunk 2 above)
Comment (steren) [inline index.html]: I left aligned the logo, which makes it aligned with the paragraph.
I removed the col-md-8.

**Comment by steren** [comment]
See updated screenshot in the PR description

**Comment by davidzchen** [comment]
Thanks, Steren! This looks great!

**Pull request merged**